torch/csrc/jit/passes/quantization/quantization_patterns.h

```diff
@@ -496,7 +496,7 @@ graph(%input_quant, %dim, %r_scale, %r_zero_point, %r_dtype):
         %r_quant = aten::quantize_per_tensor(%r, %r_scale, %r_zero_point, %r_dtype)
         return (%r_quant) )";
 
-  std::string quantized_cat = R"(
+  std::string cat_quantized_cpu = R"(
```

Comment:
this one should be reverted since it is referring to IR, not actual implementation

Reply:
Thank you for the review! 
Which should I revert `cat_quantized_cpu` only in this file or in all files?

Reply:
just this file